.pipelines/ci.yml

```diff
@@ -47,10 +51,6 @@ stages:
           python.version: '3.9'
           torch.version: 'torch==1.13.1+cpu torchvision==0.14.1+cpu torchaudio==0.13.1 --index-url https://download.pytorch.org/whl/cpu'
           ort.version: '1.14.1'
-        py38-1131:
```

Comment:
can we still keep Python version 3.8?

Reply:
yeah sounds good was planning on doing the same, had originally updated them to add python 3.12 like you did in your recent PR - just reverted most of the python versions for the moment though as it seems 3.12 might also be causing some CI problems with ort 1.17.1 on WindowsPython and MacOSPython (which we can address in a different PR).